backend/inspirehep/records/api/mixins.py

```diff
@@ -13,7 +14,94 @@
 LOGGER = structlog.getLogger()
 
 
-class CitationMixin:
+class AuthorsPapersMapExtension:
+    def generate_entries_for_authors_in_authors_records_table(self):
+        """Generates AuthorsRecords objects table based on record data for authors"""
+
+        authors_ids_field = "authors.ids"
+        table_entries_buffer = []
+        for author in self.get_value(authors_ids_field, []):
+            if author:
```

Comment:
not needed, schema guarantees that it's non-empty.

Reply:
There might be no authors at all.

Reply:
then the `for` is an empty loop